.cirrus.yml

```
@@ -37,14 +37,22 @@ task:
       test_script:
         - flutter channel $CHANNEL
         - ./script/incremental_build.sh test
-    - name: analyze
+    - name: analyze_master
       env:
         matrix:
           CHANNEL: "master"
-          CHANNEL: "stable"
       script:
         - flutter channel $CHANNEL
         - ./script/incremental_build.sh analyze
+    ## TODO(cyanglaz):
+    ## Combing stable and master analyze jobs when integration test null safety is ready on flutter stable.
+    - name: analyze_stable
+      env:
+        matrix:
+          CHANNEL: "stable"
+      script:
+        - flutter channel $CHANNEL
+        - ./script/incremental_build.sh analyze --exclude "google_sign_in_web, connectivity_for_web, file_selector_web, google_maps_flutter_web, url_launcher_web"
```

New version that should return 0, as expected (with `-depth`):

```suggestion
        - flutter channel $CHANNEL
        - find . -depth -type d -wholename '*_web/example' -exec rm -rf {} \;
        - ./script/incremental_build.sh analyze
```

---

(Added)